test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -70,6 +71,16 @@ public void TensorFlowTransformMatrixMultiplicationTest()
             }
         }
 
+        [Fact]
+        public void TensorFlowListLayersMnistConv()
+        {
+            var model_location = "mnist_model/frozen_saved_model.pb";
+            using (var env = new TlcEnvironment(seed: 1, conc: 1))
+            {
+                var schema = TensorFlowUtils.GetModelSchema(env, model_location);
```

Comment:
>schema [](start = 20, length = 6)

It would be nice to have schema checked against the actual model information. It seems like `mnist_model/frozen_saved_model.pb` is a big model. Matrix multiplication model used in above test may be a good for this test.
It can be implemented as a new test in addition to this one.